test/zeroD/test_zeroD.cpp

```diff
@@ -0,0 +1,69 @@
+#include <cstdio>
+#include <string>
+#include "gtest/gtest.h"
+#include "cantera/thermo.h"
+#include "cantera/kinetics.h"
+#include "cantera/zerodim.h"
+
+using namespace Cantera;
+
+// This test ensures that prior reactor initialization of a reactor does
+// not affect later integration within a network. This example was
+// adapted from test_reactor.py::test_equilibrium_HP.
+TEST(ZeroDim, test_individual_reactor_initialization)
+{
+    // Initial conditions
+    double T0 = 1100.0;
+    double P0 = 1013250;
+    std::string X0 = "H2:1.0, O2:0.5, AR:8.0";
+    // Reactor solution, phase, and kinetics objects
+    std::shared_ptr<Solution> sol1 = newSolution("h2o2.yaml");
+    std::shared_ptr<ThermoPhase> gas1 = sol1->thermo();
+    std::shared_ptr<Kinetics> kin1 = sol1->kinetics();
+    gas1->setState_TPX(T0, P0, X0);
+    // Set up reactor object
+    Reactor reactor1;
+    reactor1.setKineticsMgr(*kin1);
```

Comment:
You could use `reactor1.insert(sol1);` to simplify the code here and below.

Reply:
I was actually able to update this in multiple places. Thank you for the suggestion.